### ISSUE TITLE
SDL Validation doesn't allow implementing type to have optional field arguments

### ISSUE THREAD

**Issue by CarsonF** [graphql-java/graphql-java#1932]
**Describe the bug**

```graphql
interface Vehicle {
  name: String!
}

type Car implements Vehicle {
  name(charLimit: Int = 10): String!
}
```
Since `charLimit` is optional, the spec says it still conforms to the interface.
https://github.com/graphql/graphql-spec/pull/110
https://github.com/graphql/graphql-spec/issues/401

This seems really close to #1921, maybe they could be addressed at the same time?

**Comment by felipe-gdr** [comment]
Hey @CarsonF this is actually a duplicate of another issue that was closed recently: https://github.com/graphql-java/graphql-java/issues/1897

**Comment by CarsonF** [comment]
My bad! 🤦 Glad it's already fixed!

**Comment by andimarek** [comment]
@CarsonF I con confirm this is fixed in master. See https://github.com/graphql-java/graphql-java/commit/f328040f8f41a9707d9f973cb55a48621d45dd3f

It will be released with 15.0 which comes out very soon.
